build.sbt

```diff
@@ -1,16 +1,78 @@
+import scala.collection.immutable.Seq
+
 lazy val `chatops4s` = (project in file("."))
   .settings(commonSettings)
+  .settings(
+    publish / skip := true
+  )
   .aggregate(
+    `chatops4s-core`,
+    `chatops4s-slack`,
+    `chatops4s-examples`,
     `chatops4s-discord`,
     `chatops4s-discord-example`,
   )
+lazy val `chatops4s-core` = (project in file("modules/chatops4s-core"))
+  .settings(commonSettings)
+  .settings(
+    libraryDependencies ++= Seq(
+      "org.typelevel" %% "cats-effect" % "3.6.2",
+      "org.typelevel" %% "cats-core" % "2.13.0"
+    ),
+    Test / parallelExecution := false
+  )
+
+lazy val `chatops4s-slack` = (project in file("modules/chatops4s-slack"))
+  .settings(commonSettings)
+  .settings(
+    libraryDependencies ++= Seq(
+      "com.softwaremill.sttp.client4" %% "core" % "4.0.9",
+      "com.softwaremill.sttp.client4" %% "cats" % "4.0.9",
+      "com.softwaremill.sttp.client4" %% "circe" % "4.0.9",
+      "com.softwaremill.sttp.client4" %% "httpclient-backend" % "4.0.9",
+      "io.circe" %% "circe-core" % "0.14.14",
+      "io.circe" %% "circe-generic" % "0.14.14",
+      "io.circe" %% "circe-parser" % "0.14.14",
+      "com.github.pureconfig" %% "pureconfig-core" % "0.17.9",
```

Comment:
Normally this is not a desired dependenyc for a library. Its good for application but libraries try to have as little dependencies as possible. We can leave it for now though and remove in a separate PR.